pkg/cloudprovider/aws/aws_instancegroups.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws_cloud
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+	"github.com/golang/glog"
+)
+
+// AWSCloud implements InstanceGroups
+var _ InstanceGroups = &AWSCloud{}
+
+// Implement InstanceGroups.Resize
+// Set the size to the fixed size
+func (a *AWSCloud) Resize(instanceGroupName string, size int) error {
+	request := &autoscaling.UpdateAutoScalingGroupInput{}
```

Comment:
Any reason you do the initialization this way instead of:
```
request := &autoscaling.UpdateAutoScalingGroupInput{
	AutoScalingGroupName: aws.String(instanceGroupName),
	MinSize:              aws.Long(int64(size)),
	MaxSize:              aws.Long(int64(size)),
}
```
?